building-view-counter/index030.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<!--
+    Now we can see the text in our web page but we haven't populated the count yet.
```

Comment:
This could be broken up into two steps. The first could instruct the learner to wrap the text in a span element, and the second could be adding the rule to the CSS file.

The first step could read:

Now we can see the text, but we haven't populated the count yet.
To get the count, we will send an AJAX request to the server, but it will take a moment to receive a response.
We should hide the text until we get the count from the server.
Wrap the text you just added in a `<span>` element and give it the `id` "message".

The next step should appear in the CSS file and could read:

Create a new rule targeting the "message" `id` and set `display` to `none`.